### PR TITLE
Feature/small logos

### PR DESCRIPTION
**Pull request by liaa2**
Add sponsors logos
Screen Shot 2020-09-21 at 8 53 31 am

### CODE REVIEW

**Review by phazel — APPROVED** [review]

File: project.html [hunk 1]
```diff
@@ -120,3 +120,18 @@ <h4 class="reusable-title reusable-font-semi-bold">Partner and supporting organi
     </div>
   </div>
 </div>
+
+<div class="image-container component-individual data-table-margin-bottom">
+  <h2 class="data-title title-padding-bottom">Logos</h2>
+  <p class="data-subtitle">
+    Single-column display of sponsoring organisation logos and names. 
+  </p>
+  <div class="schema-page-image">
```
Comment (ragkoushik) [inline project.html:129]: Logos seem a bit squished as the width of the logo is wider than the space on the UI. Unsure if that'd be a problem?

Comment (liaa2) [reply project.html:129]: The easiest solution is to get a smaller logo😅, but yes it's a concern, let me see if I can adjust it. We may need more small logos from them

Comment (liaa2) [reply project.html:129]: @ragkoushik @phazel I've updated the image, one with specific height one without, which one you feel is better?
Screen Shot 2020-09-22 at 5 36 38 pm
With specific height the image stretched a bit:
Screen Shot 2020-09-22 at 5 40 23 pm

Comment (ragkoushik) [reply project.html:129]: One without height. If you fix the height the aspect ratio of the image will change as column widths are fixed.

Comment (liaa2) [reply project.html:129]: yeah I didn't include the height in the PR, it looks lower compares to other 2 tho. Will leave it as it is let Steph have a look. Enjoy your evening!

**Pull request merged**